docs/blog/20230501-Daeraxa-MayUpdate.md

```diff
@@ -0,0 +1,168 @@
+
+The first change adds shorthand string author assignment in the `package.json` file of a package. The [npm package specification(https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)] allows the three author properties to be combined into a single string e.g.
+
+```js
+{
+  "author": "Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"
+}
+```
+
+Previously if this was not populated fully with all three options then the author simply wouldn't display. This fixes that problem by allowing the optional fields to be omitted (actually this is taken a step further in our implementation - even `name` can be optional)
+
+In a similar vein we also now support [shortcut syntax](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository) for the `repository` field of the `package.json`. For example writing `authorName/repoName` wasn't working so we have provided support for this.
+
+We noticed that many package authors would link to other community packages in their readmes in order to show other packages that work with or complement their package. Unfortunately as these point to the old `atom.io` website it means that every single link simply redirects to the [sunsetting Atom](https://github.blog/2022-06-08-sunsetting-atom/) page. This is really not helpful as it entirely breaks the embedded links, sure there are workaround, people could look at the link and work out what packages it links to but this is a hassle when all you want to do is click a link.
+
+<img src="./assets/sunsetting.png"/>
+
+Therefore in order to solve this a feature has been added that takes these `https://atom.io/packages/...` links and transforms them to `https://web.pulsar-edit.dev/packages/...` so that the author's original intent for package linking is not lost. We did have a [conversation](https://github.com/pulsar-edit/package-frontend/issues/95) as to whether it was "right" to make these changes but we reasoned that:
+
+1. these links are all effectively "dead" so currently serve no useful purpose and
+2. they are _already_ being redirected from the original link.
+
+We feel that this is an entirely benign change even though it does redirect from the author's originally intended link to another.
+
+Our next change also involves dead Atom links, in this case links to the Atom Flight Manual which has been taken down along with the rest of the `atom.io` sites. Again we are performing a redirection here but this time to the archived version of the website found on [Internet Archive](https://web.archive.org/web/20221215003438/https://flight-manual.atom.io/). This keeps the correct link handling and means that the documentation the package author was intending to link to is still available.
```

Comment:
Also, I didn't realize until now that the “archived version” we were redirecting to was the one from the Wayback Machine. There's also [the version archived here](https://atom-flight-manual-archive.github.io/); it ought to be quite a bit faster.

Reply:
This was actually a choice on my end about where the redirects would go on the frontend.

While I'm aware of that other archive, I thought it'd make more sense to redirect to an official source, rather than a community member hosting the page. Especially if it's only there to handle the odd page or two being redirected to it

Reply:
I did think the same (really this would be a discussion for the actual PR as this is just describing it) but the internet archive one is likely to be stable where the other one could potentially be taken down without notice.